src/Symfony/Component/Console/Tests/Helper/LegacyTableHelperTest.php

```diff
@@ -34,7 +34,7 @@ protected function tearDown()
     }
 
     /**
-     * @dataProvider testRenderProvider
+     * @dataProvider renderProvider
```

Comment:
This change is already in the `2.8` because of #22379. It looks like I missed it for the `2.7` branch in #22066.